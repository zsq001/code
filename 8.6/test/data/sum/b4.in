100 100
1260637 1141764 720282 2234437 1646402 1281637 107914 2809041 1048551 2368687 1272137 2404226 121371 985233 1342195 239275 1199591 393538 1344332 2957475 2402409 2325918 569207 2148101 1832756 3038782 1003686 1515632 2273674 2347207 2394372 1782829 3239045 342427 1051394 2276128 3023754 995210 1861787 3236626 1478149 2753626 1765348 1581462 2950627 3203492 2628576 529951 163350 389465 1773840 3046435 854092 1284370 2731193 2567776 2714881 1295697 3187753 80018 1233300 3162641 3191214 2423586 1305922 698270 676823 777238 3232244 1692446 302173 2394710 1818732 2617921 3214907 446613 2756891 2792694 560178 2881382 2959638 2425112 3200638 79711 1081385 1448698 6987 1129566 1653074 1802715 1610662 1980222 247628 147870 2634133 3146122 1549269 1098983 123026 1658471 
1 88 92 905413
2 1 45
1 13 75 2001911
2 79 79
1 13 61 73663
2 67 94
1 52 60 2747490
2 29 78
1 36 48 3006397
2 62 71
1 35 76 302598
2 41 61
1 4 28 1223742
2 41 73
1 1 97 1235035
2 58 87
1 48 54 1824231
2 6 80
1 55 65 1848801
2 42 49
1 3 92 730452
2 72 99
1 65 100 1137497
2 45 76
1 37 86 3192055
2 15 31
1 9 49 1931801
2 64 83
1 16 92 1080429
2 19 73
1 59 100 2616862
2 17 88
1 10 66 421267
2 26 93
1 28 66 1567364
2 6 26
1 47 72 2012573
2 84 92
1 2 48 3145231
2 50 97
1 51 54 2919093
2 72 96
1 53 58 2120736
2 13 66
1 3 10 2987802
2 40 96
1 18 61 340542
2 67 90
1 67 73 2745875
2 47 78
1 47 83 131741
2 47 78
1 61 70 2713690
2 7 66
1 11 77 1883187
2 12 23
1 64 86 586024
2 46 87
1 7 61 2527178
2 55 70
1 22 41 2507456
2 1 4
1 58 61 3161637
2 64 88
1 27 88 2690134
2 23 100
1 72 76 2579139
2 16 44
1 67 79 28553
2 10 44
1 9 50 1195144
2 19 22
1 59 66 1896033
2 24 33
1 72 76 2432562
2 13 66
1 59 82 3266785
2 5 89
1 18 76 2630427
2 6 56
1 15 70 1416077
2 6 14
1 42 75 3130216
2 36 48
1 86 98 2230121
2 67 78
1 8 20 2381530
2 44 100
1 61 86 3206730
2 4 89
1 25 58 1621071
2 21 37
1 42 90 111737
2 21 62
1 22 72 2251431
2 36 56
1 37 94 44424
2 44 44
1 84 100 1307043
2 8 59
