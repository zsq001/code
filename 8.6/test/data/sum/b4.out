889464351
73855735
168346625
547012185
485689226
198426704
584989326
279589482
474421191
693111622
51930501
313708931
277773742
454957508
292462888
712844710
474665997
6686225
732331590
369464476
867688950
228028570
150464759
351693499
926762632
889581371
74211808
749080786
976480445
918870923
943967145
333971493
440731132
793182752
710980177
993001501
851243352
424761919
18249765
400432866
297404138
664985552
672008478
887577511
180632268
56992810
328231288
513280207
897728776
484988456
